src/mid.cpp

```diff
@@ -114,6 +114,7 @@ typedef unsigned int   uint32_t;
 // File types
 #define FILE_LUCAS      1
 #define FILE_MIDI       2
+#define FILE_MIDItype1  7 // Multiple track file format
```

Comment:
Can you update `FILE_MIDI` to be `FILE_MIDItype0` as well so it's clear what the difference is?

Reply:
Please note that it does not detect if it is type0 or type1 until rewind() has been called if I remember the code correct, since it needs to parse more header data than is done in the initial detection routine.

Reply:
I have added a new member to the class for this, as the code looks more clear this way.